gtfs-realtime-validator-webapp/pom.xml

```diff
@@ -150,7 +150,7 @@
         <dependency>
             <groupId>edu.usf.cutr</groupId>
             <artifactId>gtfs-realtime-validator-lib</artifactId>
-            <version>1.0.0-SNAPSHOT</version>
+            <version>${parent.version}</version>
```

Comment:
Have you taken a look at https://maven.apache.org/maven-ci-friendly.html?

That has some specific directions about what is and isn't allowed for variable names.